DOCUMENTATION.md

```diff
@@ -2516,7 +2517,7 @@ await map.clear();
 
 The Entry Listener is used by the Hazelcast `MultiMap` and `ReplicatedMap`.
 
-You can listen to map-wide or entry-based events by using the functions in the `EntryListener` interface. Every function type
+You can listen to map-wide or entry-based events using the `EntryListener` interface functions. Every function type
```

Comment:
I think EntryListener interface functions does not sound good. There are some fields that are expected to be functions in the interface. I think you should revert the change except deleting `by`.